tas-with-installed-tile/app/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	log.SetFormatter(&log.TextFormatter{DisableTimestamp: true, DisableLevelTruncation: true})
+	log.SetLevel(log.DebugLevel)
+
+	log.Infof("Environment:\n%s\n\n", strings.Join(os.Environ(), "\n"))
+
+	http.HandleFunc("/", func(res http.ResponseWriter, req *http.Request) {
+		log.Infof("Handling request\n")
+		res.WriteHeader(200)
+
+		var secret = "[envvar not found]"
+		if secretValue, ok := os.LookupEnv("DEMO_SECRET"); ok {
+			secret = secretValue
+		}
+
+		_, _ = fmt.Fprintf(res, `
```

Comment:
My Golang is a bit rusty... but is there a reason to use the blank identifiers `_, _` here?
Could this be:
```
fmt.Fprintf(res,
.
.
.
```

Reply:
It's possible to ignore them but I normally add the blank identifiers to make explicit that we're ignoring errors.